.github/workflows/provisioning-tests.yml

```
@@ -51,7 +51,10 @@ jobs:
           docker compose --version
 
       - name: free up disk space
-        run:  sudo apt remove --purge -y ghc-* azure-cli google-cloud-sdk hhvm llvm-* dotnet-* powershell mono-* php* ruby*
+        # 2023-09-28: google-cloud-sdk removed from this list because it was intermittently
+        #   unavailable as an apt package to remove, and might be migrating to snap. If more
+        #   disk space is needed, see if the snap is installed, and remove that.
+        run:  sudo apt remove --purge -y ghc-* azure-cli hhvm llvm-* dotnet-* powershell mono-* php* ruby*
```

@timmc-edx: Is there an alternative that breaks it out on to its own line and swallows errors?

---

Yes, but I'm not sure it's worth it unless we see out-of-space issues again. And in that case, I'd first try removing the snap. But I don't have a lot of experience with snap, so this would take more research.

Interestingly, in February it sounded like GitHub might be planning on [not pre-installing this apt package anymore](https://github.com/actions/runner-images/issues/7101#issuecomment-1431174447) partly due to the age of the package; maybe Ubuntu itself is removing the apt package as well. If that's the case, GitHub _probably_ isn't replacing it with the snap package either.